move-vm-backend/build.rs

```diff
@@ -0,0 +1,19 @@
+use std::process::Command;
+
+fn main() {
+    // Tell Cargo that if the given file changes, to rerun this build script.
+    println!("cargo:rerun-if-changed=../deposit/sources/deposit.move");
+    // SMOVE build our deposit code
+    Command::new("move")
```

Comment:
So we are building with plain `move` or `smove` - comment says `smove` but `move` is called 😸

Reply:
I've raised this with @Rqnsom as `smove` does not work with `-p` atm...